modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,7 +3,109 @@
 // * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_content-type: PROCEDURE
+
+###################################################
 [id="ztp-adding-new-content-to-gitops-ztp_{context}"]
+= Adding custom content to the {ztp} pipeline
+
```

Comment:
Need some prereqs here to set the context for the procedure, eg., you must have a site config repo, ArgoCD application configured, etc.

Reply:
I am not getting the reason behind to set pre-requisites here as there is no change in implementation design. we are proposing another way how to source-crs with customization as per end-users requirements.

Reply:
ok, we'll leave as is so.